plugins/restful/RestfulDataProviderEFQ.php

```diff
@@ -140,7 +140,9 @@ protected function queryForListSort(\EntityFieldQuery $query) {
 
     foreach ($sorts as $public_field_name => $direction) {
       // Determine if sorting is by field or property.
-      $property_name = $public_fields[$public_field_name]['property'];
+      if (!$property_name = $public_fields[$public_field_name]['property']) {
```

Comment:
well, probably as a follow up, we should throw an excpetion, if ``getColumnFromProperty`` returned FALSE (as some EMW properties may be "computed"). I'll open an issue for that.